models/intermediate/int_ad_reporting__account_report.sql

```diff
@@ -0,0 +1,118 @@
+with prep_standardized_union as (
+
+    {{ dbt_utils.union_relations(
+        relations=[
+            ref('facebook_ads__account_report'),
+            ref('google_ads__account_report'),
+            ref('microsoft_ads__account_report'),
+            ref('twitter_ads__account_report')      
+            ],
+        source_column_name='platform',
+        include=['date_day',
+                'account_id', 
+                'account_name', 
+                'clicks', 
+                'impressions', 
+                'spend']) }}
+), 
+
+prep_standardized_union_platform_rename as (
+
+    select 
+        cast(date_day as DATE) as date_day,
```

Comment:
I am not sure if `DATE` is universal across platforms. Is there a util we should be using instead?